pkg/envoy/lds/connection_manager.go

```diff
@@ -1,21 +1,64 @@
 package lds
 
 import (
+	"path/filepath"
+
+	envoy_config_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
 	xds_route "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
+	xds_wasm "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/http/wasm/v3"
 	xds_hcm "github.com/envoyproxy/go-control-plane/envoy/extensions/filters/network/http_connection_manager/v3"
+	xds_wasm_ext "github.com/envoyproxy/go-control-plane/envoy/extensions/wasm/v3"
 	"github.com/envoyproxy/go-control-plane/pkg/wellknown"
 
+	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/wrappers"
 
 	"github.com/openservicemesh/osm/pkg/configurator"
 	"github.com/openservicemesh/osm/pkg/constants"
 	"github.com/openservicemesh/osm/pkg/envoy"
+	"github.com/openservicemesh/osm/pkg/featureflags"
 )
 
 const (
 	statPrefix = "http"
 )
 
+func getStatsWASMFilter() (*xds_hcm.HttpFilter, error) {
+	wasm := &xds_wasm.Wasm{
+		Config: &xds_wasm_ext.PluginConfig{
+			Name: "stats",
+			Vm: &xds_wasm_ext.PluginConfig_VmConfig{
+				VmConfig: &xds_wasm_ext.VmConfig{
+					Runtime: "envoy.wasm.runtime.v8",
+					Code: &envoy_config_core_v3.AsyncDataSource{
+						Specifier: &envoy_config_core_v3.AsyncDataSource_Local{
+							Local: &envoy_config_core_v3.DataSource{
+								Specifier: &envoy_config_core_v3.DataSource_Filename{
```

Comment:
Instead of using a file from a ConfigMap, why don't we just pass this to osm-controller directly and use `inline_bytes` as the DataSource? This would void the need for a ConfigMap.

Reply:
I think that would work. I'll try it to make sure.